aspnetcore-1.0.yaml

```diff
@@ -1,5 +1,6 @@
 steps:
 - name: 'gcr.io/gcp-runtimes/aspnetcorebuild-1.0:latest'
+  args: [ '-r', 'gcr.io/google-appengine/aspnetcore:1.0' ]
```

Comment:
nit: I think it'd be clearer to use the long form here–it makes it pretty obvious what '-r' means.